http/vibe/http/server.d

```diff
@@ -633,6 +642,15 @@ final class HTTPServerSettings {
 	*/
 	string hostName;
 
+	/** Provides a way to ban and unban network addresses and reduce the
+		impact of DOS attack.
+
+		If any of the ban managers bans an address, then all incoming requests
+		from that address will be rejected.
+	*/
+	alias IsBannedDlg = bool delegate (ref in NetworkAddress) @safe nothrow;
+	IsBannedDlg[string] isBannedDlgs;
```

Comment:
Why is this an AA ? A simple delegate would do.

Reply:
My idea was to allow multiple different types of BanManagers and if any of them would drop the connection, then it will be dropped. Should I just allow 1 BanManager?

Reply:
One delegate is enough. If a user needs multiple delegates, they can just wrap them in one delegate that do the iteration.